### DISCUSSION TITLE
Confirmation: Custom Map link works perfectly

### DISCUSSION THREAD

**Discussion by jpretired** [vesta-webtrees-2-custom-modules/vesta_personal_facts#87] [Show and tell]
This is a note to Custom Map link through Control panel > Modules > Vesta Facts and events > Preferences > Map links > Custom Map provider:
My settings:
Name: "Mapy.cz"
URI template: `https://mapy.cz/zakladni?x={long}&y={lati}&z=15&source=coor&id={long},{lati}`
The last part of URI starting with "&source=..." enables display of place pointer on the map.

![image](https://user-images.githubusercontent.com/350656/192993558-c875e274-77a9-4676-bccb-29d71686bb0c.png)

The above picture shows part of map called by https://en.mapy.cz/zakladni?x=16.9150028&y=49.6001319&z=15&source=coor&id=16.9150028,49.6001319

**Comment by ric2016** [comment]
Thanks for the note - I have updated the example url in the control panel.
